android/app/src/main/java/com/reactnativenavigation/activities/BaseReactActivity.java

```diff
@@ -355,6 +389,20 @@ public void invokeDefaultOnBackPressed() {
         super.onBackPressed();
     }
 
+    protected void setupDrawer(Screen screen, Drawer drawer, int drawerFrameId, int drawerLayoutId) {
```

Comment:
Nevermind me, it makes sense.  They're different layouts, so they will generate different layout IDs.